build.assets/tooling/cmd/difftest/stringset.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+// Void represents an empty structure
+type Void struct{}
+
+// StringSet represents a set of strings
+type StringSet map[string]Void
```

Comment:
```suggestion
// Void represents an empty structure
type Void struct{}

// StringSet represents a set of strings
type StringSet map[string]struct{}
```

I would expect most devs to know that `struct{}` is used when you don't care about the value. There's no need to export this type and make it part of the package's public API, since it really has nothing to do with a string set.